opentelemetry-sdk/tests/logs/test_handler.py

```diff
@@ -79,6 +79,19 @@ def test_log_record_emit_noop(self):
             logger.warning("Warning message")
         handler_mock._translate.assert_not_called()
 
+    def test_log_flush_noop(self):
+
+        no_op_logger_provider = NoOpLoggerProvider()
+        no_op_logger_provider.force_flush = Mock()
+
+        logger = get_logger(logger_provider=NoOpLoggerProvider())
```

Comment:
I think you're not using the mock here.

```suggestion
        logger = get_logger(logger_provider=no_op_logger_provider)
```

Reply:
right, fixing...